api/src/main/java/org/datacleaner/metadata/ColumnMeaning.java

```diff
@@ -35,7 +35,7 @@
 
     OTHER("Disregard", "Other", "Nothing"),
 
-    KEY_PRIMARY("Primary Key", "Record ID", "ID", "Identifier", "key", "PKID", "Record key"),
+    KEY_PRIMARY("Primary Key", "ID", "Record ID", "Identifier", "key", "PKID", "Record key"),
```

Comment:
Hmm coincidence I think. I first tried another approach but changed my mind. Think this is a left-over of that attempt :-)